editor/src/components/editor/use-default-collapsed-views.tsx

```diff
@@ -46,13 +46,15 @@ export function useDefaultCollapsedViews() {
     })
 
     if (newCollapsedViews.length > 0) {
-      // 1. add the new collapsed views
-      dispatch([addCollapsedViews(newCollapsedViews.map(EP.fromString))])
-      // 2. store the new added views to the previous ones
-      previouslyCollapsed.current = newCollapsedViews.reduce(
-        (acc, path) => ({ ...acc, [path]: true }),
-        previouslyCollapsed.current,
-      )
+      queueMicrotask(() => {
```

Comment:
This is an additional place where the rule caught a direct call to `dispatch`, I wrapped the entire logic in `queueMicrotask` since I didn't want to separate the two steps